Core.Collectors/IO/AzureBlobRecordWriter.cs

```diff
@@ -31,20 +31,20 @@ public class AzureBlobRecordWriter<T> : RecordWriterCore<T> where T : FunctionCo
                                      ITelemetryClient telemetryClient,
                                      T functionContext,
                                      ContextWriter<T> contextWriter,
-                                     string storageConnectionEnvironmentVariable = "AzureWebJobsStorage",
+                                     string storageAccountNameEnvironmentVariable = "StorageAccountName",
```

Comment:
>string storageAccountNameEnvironmentVariable = "StorageAccountName", [](http://example.com/codeflow?start=37&length=68)

This normally points to (e.g., in GitHub) to the same storage account the functions are working in. Does this mean that we need to change this to the name of the storage account for GH deployments as well? If yes, fine, just checking.

Reply:
Yes, that is correct. I will have to do a GitHub Deploy later